tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -366,13 +372,9 @@ def emit_view_lambda(f: NativeFunction, unpacked_bindings: List[Binding]) -> str
                 arg=arg, val=arg_value, default="0"
             )
             updated_unpacked_args.append(arg_value)
-        elif (
-            arg == "nested_size_" or arg == "nested_strides_" or arg == "offsets_"
-        ) and arg_type == ConstRefCType(BaseCType(tensorT)):
-            # [NOTE] [Nested Arg Types]
-            # This is temporary. Nested tensors will be migrating to use SymInts and
-            # nested_size and nested_strides will no longer be tensors.
-            updated_unpacked_args.append(arg[:-1])
+        elif arg_type == ConstRefCType(BaseCType(tensorT)):
+            # NB: Since we're not actually unpacking, just use the normal arg name.
+            updated_unpacked_args.append(undo_unpacked_name(arg))
```

Comment:
@albanD is this okay or is there a better way to fix this? I was confused by the need for this workaround until I found that the unpacking logic is discarded when we generate the view_func lambda.